push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/InternalUnifiedPushMessage.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.aerogear.unifiedpush.message;
```

Comment:
Let's also move the `clientIdentifier` into this class:
https://github.com/idelpivnitskiy/aerogear-unifiedpush-server/blob/AGPUSH-1381/push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/UnifiedPushMessage.java#L83